src/Sulu/Bundle/ContentBundle/Resources/js/stores/WebspaceStore/WebspaceStore.js

```diff
@@ -7,15 +7,31 @@ class WebspaceStore {
 
     webspacePromise: Promise<Object>;
 
-    loadWebspaces(): Promise<Array<Webspace>> {
+    sendRequest(): Promise<Object> {
         if (!this.webspacePromise) {
             this.webspacePromise = Requester.get(this.baseUrl);
         }
 
-        return this.webspacePromise.then((response: Object) => {
+        return this.webspacePromise;
+    }
+
+    loadWebspaces(): Promise<Array<Webspace>> {
+        return this.sendRequest().then((response: Object) => {
             return response._embedded.webspaces;
         });
     }
+
+    loadWebspace(webspaceKey: string): Promise<Webspace> {
+        return this.sendRequest().then((response: Object) => {
+            for (let webspace of response._embedded.webspaces) {
```

Comment:
`const` should also work here: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of